app/src/main/java/io/scalac/degree/android/fragment/TalksFragment.java

```diff
@@ -250,6 +272,10 @@ private void fillHolder(ViewHolder holder, int position) {
                 holder.imageButtonNotify.setColorFilter(notscheduledStarColor);
             }
             holder.imageButtonNotify.setTag(position);
+
+            final String trackIconUrl = tracksDownloader.getTrackIconUrl(slotModel.talk.track);
```

Comment:
ta metode bym rozbil na mniejsze, a `holder.text3` to nie jest najszczesliwsza nazwa pola :)